x-pack/legacy/plugins/siem/public/components/ml/api/throw_if_not_ok.ts

```diff
@@ -4,19 +4,50 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { has } from 'lodash/fp';
+
+import * as i18n from './translations';
+
 export interface MessageBody {
   error?: string;
   message?: string;
   statusCode?: number;
 }
 
+export interface MlErrorMsg {
```

Comment:
May want to rename this as `MlStartJobError` as it looks like this is used explicitly for the `startDatafeeds` API call?

Reply:
Just commenting as I saw `started` wasn't optional, which then led me to trace further and see where this was being used exactly.

Reply:
Yea, I will rename it to be `MlStartJobError`